lib/tasks/data.rake

```diff
@@ -0,0 +1,7 @@
+namespace :data do
+  task populate_npq_profiles: :environment do
+    NPQProfile.where.not(id: ParticipantProfile::NPQ.select(:id)).find_each do |profile|
+      profile.send(:update_participant_profile)
```

Comment:
Why `.send(:update_participant_profile)` rather than `.update_participant_profile`?

Reply:
It is a private method. I could have made it public, but it is to go away very soon together with this task :P